src/shared/api/genomeNexusClientInstance.ts

```diff
@@ -2,10 +2,12 @@ import { GenomeNexusAPI } from 'genome-nexus-ts-api-client';
 
 async function checkVersion(client: GenomeNexusAPI) {
     const versionResp = await client.fetchVersionGET({});
-    if (parseInt(versionResp.version.split('.')[0]) !== 1) {
+    if (
+        parseInt(versionResp.genomeNexus.database.version.split('.')[0]) !== 1
+    ) {
         console.error(
             'Expected version of Genome Nexus to be 1.x.y, but found: ' +
-                versionResp.version
+                versionResp.genomeNexus.database.version
```

Comment:
```suggestion
                versionResp.genomeNexus.server.version
```
Probably should be `server` too